frontend/src/components/Statssection.tsx

```diff
@@ -0,0 +1,91 @@
+import { Grid } from '@mui/material';
+import { StatsService } from 'src/services/openapi';
+import type { Statistics } from 'src/services/openapi';
+import React, { useEffect, useState } from 'react';
+import useMediaQuery from '@mui/material/useMediaQuery';
+
+interface statsType {
+  text: string;
+  count: number;
+  last_month_count: number;
+}
+
+const StatsUi = (props: statsType) => {
+  return (
+    <Grid item xs={4}>
+      <span style={{ fontSize: '12px' }}>{props.text}</span> <br />
```

Comment:
Also, try to use the props `sx`instead of `style`
There is another active PR that dealing with this issue. So we try our best not to introduce anomaly into the code